File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    const arr = s.slice(2, -2).split('},{').sort((a, b) => a.length - b.length);
    const answer = [];
    for(const item of arr) {
       const numbers = item.split(',');
        for(const num of numbers) {
            if(!answer.includes(Number(num))) answer.push(Number(num));
        }
    }
    
    return answer;
}

File: 프로그래머스/2/64065. 튜플/튜플.py
def solution(s):
    arr = sorted(s[2:-2].split('},{'), key=lambda x: len(x))
    answer = []
    t_set = set()
    for t in arr:
        part = t.split(',')
        for c in part:
            if int(c) not in t_set:
                t_set.add(int(c))
                answer.append(int(c))
    return answer    
    